### ISSUE TITLE
help request: Rejected message for request-validation plugin

### ISSUE THREAD

**Issue by pavlenkodev** [apache/apisix#10678]
### Description

In the request-validation plugin, you can add a rejected message using your own text. If you do not add your own text (rejected message), validation errors will be displayed.

Is it possible to make my text (rejected message) output and the detected validation errors added to it?

### Environment

APISIX version 3.5:
etcd version 3.4.15
APISIX Dashboard version 3.01

**Comment by kayx23** [comment]
This would perhaps require serverless plugin. @shreemaan-abhishek could advise a bit more.

**Comment by pavlenkodev** [comment]
@shreemaan-abhishek Hello! Could you recommend something?

**Comment by shreemaan-abhishek** [comment]
I don't think this is possible, but it seems this can be simple to implement. @pavlenkodev would you like to contribute this?

**Comment by pavlenkodev** [comment]
Unfortunately, my knowledge of the lua language is not enough to implement this task.  Therefore, I cannot promise that I will do it. But maybe I'll try later. At the moment, you should not appoint me as a performer. I will let you know if I start completing the tasks

Do you have a guide for deploying apisix locally somewhere?

**Comment by smileby** [comment]
@pavlenkodev  You can take a look at this
https://apisix.apache.org/docs/apisix/building-apisix/

**Comment by smileby** [comment]
@pavlenkodev Hi, Do you have any examples of what is expected?  For example, how to configure, input parameters, expected results

**Comment by pavlenkodev** [comment]
I apologize for taking so long to answer your question. As I see possible extensions of this function.
1. If possible, you can add a response format for the rejected message (json, text, xml etc) At the moment, the response in the case of rejected message is plain/text
2. It is possible to organize validation errors into text that could be supplied using the keyword $errors, for example.
```
rejected_msg : {myResponse : $errors}
```

It seems to me that the proposed options will cover most of the rejected_msg use cases, if they can be implemented and if you like them

**Comment by pavlenkodev** [comment]
> @pavlenkodev Hi, Do you have any examples of what is expected? For example, how to configure, input parameters, expected results

I apologize for taking so long to answer your question. As I see possible extensions of this function.
1. If possible, you can add a response format for the rejected message (json, text, xml etc) At the moment, the response in the case of rejected message is plain/text
2. It is possible to organize validation errors into text that could be supplied using the keyword $errors, for example.
```
rejected_msg : {myResponse : $errors}
```

It seems to me that the proposed options will cover most of the rejected_msg use cases, if they can be implemented and if you like them

**Comment by Baoyuantop** [comment]
Closed due to prolonged lack of progress. If you're still interested in this feature, please let us know.
